### ISSUE TITLE
Update this repo

### ISSUE THREAD

**Issue by lamogui** [philandstuff/portmidi#1]
Do you plan to update this repo (I see on sourceforge that the most recent changes are from 2015 but here it's 2012)
Because I wanted to use your repo as git submodule for my dependancies.

Thx in advance for any answer.

**Comment by philandstuff** [comment]
No, to be honest I'd forgotten I had this repo, sorry.  I'm not doing any
work with portmidi any more.
